doc/quickstart/python.md

```diff
@@ -32,7 +34,7 @@ For information on that, follow the instructions in the
 
 ## Simple mlpack quickstart example
 
-As a really simple example of how to use mlpack from Python, let's do some
+As a really simple example of how to use MLpack from Python, let's do some
```

Comment:
Use of 'mlpack' is more common across the file and (fairly if not even entirely) consistent so I would change this back.